src/test/java/com/thealgorithms/strings/IsomorphicTest.java

```diff
@@ -1,31 +1,21 @@
 package com.thealgorithms.strings;
 
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 
-import org.junit.jupiter.api.Test;
+import java.util.stream.Stream;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
 
 public final class IsomorphicTest {
-    private IsomorphicTest() {
-    }
-
-    @Test
-    public static void main(String[] args) {
-        String str1 = "abbbbaac";
-        String str2 = "kffffkkd";
-
-        String str3 = "xyxyxy";
-        String str4 = "bnbnbn";
 
-        String str5 = "ghjknnmm";
-        String str6 = "wertpopo";
-
-        String str7 = "aaammmnnn";
-        String str8 = "ggghhhbbj";
+    @ParameterizedTest
+    @MethodSource("isomorphicProvider")
+    public void testCheckStrings(String str1, String str2, Boolean expected) {
+        assertEquals(expected, Isomorphic.checkStrings(str1, str2));
+    }
 
-        assertTrue(Isomorphic.checkStrings(str1, str2));
-        assertTrue(Isomorphic.checkStrings(str3, str4));
-        assertFalse(Isomorphic.checkStrings(str5, str6));
-        assertFalse(Isomorphic.checkStrings(str7, str8));
+    private static Stream<Arguments> isomorphicProvider() {
+        return Stream.of(Arguments.of("abbbbaac", "kffffkkd", Boolean.TRUE), Arguments.of("xyxyxy", "bnbnbn", Boolean.TRUE), Arguments.of("ghjknnmm", "wertpopo", Boolean.FALSE), Arguments.of("aaammmnnn", "ggghhhbbj", Boolean.FALSE));
```

Comment:
```suggestion
        return Stream.of(Arguments.of("", "", Boolean.TRUE), Arguments.of("", "a", Boolean.FALSE), Arguments.of("aaa", "aa", Boolean.FALSE), Arguments.of("abbbbaac", "kffffkkd", Boolean.TRUE), Arguments.of("xyxyxy", "bnbnbn", Boolean.TRUE), Arguments.of("ghjknnmm", "wertpopo", Boolean.FALSE),
            Arguments.of("aaammmnnn", "ggghhhbbj", Boolean.FALSE));
```

Using this opportunity, we can add the _missing_ test cases (cf. the coverage report):
- inputs with different length (not covered at all),
- _all time classics_ - some empty inputs